thundra/integrations/requests.py

```diff
@@ -30,20 +30,41 @@ def _parse_http_url(self, url):
             url_dict['path'] = parsed_url.path
             url_dict['query'] = parsed_url.query
             url_dict['host'] = parsed_url.netloc
+
+            normalized_path = self.get_normalized_path(parsed_url.path)
+            url_dict['operation_name'] = parsed_url.hostname + normalized_path
+
             url_dict['url'] = parsed_url.hostname + parsed_url.path
         except Exception:
             pass
         return url_dict
-    
+
+    def get_normalized_path(self, url_path):
+        path_depth = config.http_integration_url_path_depth()
```

Comment:
Are we sure that protocol part (`"http://` or `"https://`) is not exist in the `url_path `? If not, can we trim it here?

Reply:
Yes we are, url_path is only path (not with scheme or hostname).